exchanges/bithumb/bithumb.go

```diff
@@ -143,19 +144,27 @@ func (b *Bithumb) GetTradablePairs() ([]string, error) {
 //
 // symbol e.g. "btc"
 func (b *Bithumb) GetTicker(symbol string) (Ticker, error) {
-	response := Ticker{}
-	path := fmt.Sprintf("%s%s%s", b.APIUrl, publicTicker, common.StringToUpper(symbol))
+	type Response struct {
+		ActionStatus
+		Data Ticker `json:"data"`
+	}
+
+	response := Response{}
+	path := fmt.Sprintf("%s%s%s",
+		b.APIUrl,
+		publicTicker,
+		strings.ToUpper(symbol))
 
 	err := b.SendHTTPRequest(path, &response)
 	if err != nil {
-		return response, err
+		return response.Data, err
 	}
 
-	if response.Status != noError {
-		return response, errors.New(response.Message)
+	if response.Status != "0000" {
```

Comment:
The API was returning ints instead of strings for a passing moment last night that's why I changed, will change back because strings are being returned now.